beta-20170112.md

```diff
@@ -20,13 +20,6 @@ Get future release notes emailed to you:
     </script>
 </div>
 
-{{site.data.alerts.callout_danger}}
```

Comment:
@jess-edwards or @swatikumar16, can you remove this warning from the corresponding release notes in the forum as well?